api/tests/test_songs/test_songs_delete.py

```diff
@@ -0,0 +1,28 @@
+import pytest
```

Comment:
Il manque une vérification que le morceau supprimé n'existe bien plus.

Reply:
Fait — grâce à `ObjectDoesNotExist`… ;-)

Reply:
à la place d'un "is None".

Reply:
;P

Reply:
Note que si j'avais pas levé l'exception, j'aurais pas pu utiliser les error handlers de Falcon, et on aurait dû répéter la logique de "si pas trouvé, envoyer un 404" dans toutes les vues correspondantes. Alors que là, c'est 6 lignes maxi